set.mm

```diff
@@ -743770,7 +743770,7 @@ ordered pairs of real numbers (the cartesian product of the real
        less than the first coordinate of the other point, or the first
        coordinates of both points are equal and the second coordinate of the
        first point is less than the second coordinate of the other point:
-       (a,b) <_ (x,y) :<-> ( a<x \/ ( a=x /\ b<_y) ).  (Contributed by AV,
```

Comment:
Wouldn't the better solution be to use "math mode" here?
I.e. put the formula within back tick escapes, and space out each symbol.

Reply:
Maybe we should ask @avekens . There isn't a math symbol called `:<->` and to be honest I'm not completely sure how well the concepts here map to set.mm notions of ordered pair, no-greater-than of extended reals, etc.

Reply:
I guess this means "defined as", just like ":=" also sometimes means "defined as". 
But yes, the best is to ask @avekens .

Reply:
This should have been a semiformal definition of a lexicographical ordering. Maybe this "formula" can be replaced by `<. a , b >. <_ <. x , y >. ` iff ` ( a < x \/ ( a = x /\ b <_ y ) ) `.

Reply:
I've put that change into #3527